File: Step2-Python/2480.py
#way1
num1,num2,num3=map(int,input().split())
result=0

if num1==num2==num3:
    result=10000+num1*1000
elif num1==num2:
    result=1000+num1*100
elif num2==num3:
    result=1000+num2*100
elif num3==num1:
    result=1000+num3*100
else:
    result=max(num1,num2,num3)*100

print(result)

#way2
num=list(map(int,input().split()))

if num[0]==num[1]==num[2]:
    print(10000+num[0]*1000)
elif num[0]==num[1]:
    print(1000+num[0]*100)
elif num[1]==num[2]:
    print(1000+num[1]*100)
elif num[2]==num[0]:
    print(1000+num[2]*100)
else:
    max=0
    for i in num:
        if i>max:
            max=i 
    print(max*100)

File: Step2-C/2480.c
#define _CRT_SECURE_NO_WARNINGS
#include <stdio.h>

int max(int a, int b, int c)
{
	int maxNum = 0;
	if (a > b && a > c)
		maxNum = a;
	else if (b > c && b > a)
		maxNum = b;
	else if (c > a && c > b)
		maxNum = c;

	return maxNum;
}

int main()
{
	int num1,num2,num3;
	scanf("%d %d %d", &num1, &num2, &num3);

	// num1==num2==num3 ¿Ã∞≈ æ»µ .§æ§æ
	if (num1 == num2 && num1 == num3 && num2 == num3)
		printf("%d", 10000 + num1 * 1000);
	else if (num1 == num2)
		printf("%d", 1000 + num1 * 100);
	else if (num2 == num3)
		printf("%d", 1000 + num2 * 100);
	else if (num3 == num1)
		printf("%d", 1000 + num3 * 100);
	else
		printf("%d", max(num1, num2, num3) * 100);
	return 0;
}